api/app/tests/apps/api/v1/request_data/create_initial.json

```diff
@@ -28,7 +28,8 @@
         "email": "melder@example.com",
         "phone": "0123456789",
         "created_at": "2018-06-18T13:30:47.766420Z",
-        "updated_at": "2018-06-18T13:30:47.795622Z"
+        "updated_at": "2018-06-18T13:30:47.795622Z",
```

Comment:
aside: we should probably remove these fields (`udpated_at`, `created_at`) they should not be set by reporting.